templates/jenkins-slave-defaults.erb

```diff
@@ -9,6 +9,7 @@ JAVA=/usr/bin/java
 # arguments to pass to java
 #JAVA_ARGS="-Xmx256m"
 #JAVA_ARGS="-Djava.net.preferIPv4Stack=true" # make jenkins listen on IPv4 address
+JAVA_ARGS="<%= @java_args -%>"
```

Comment:
It would be better to not declare the variable unless it is defined.  It is possible [although unlikely] that the user would want the value of `JAVA_ARGS` to be inherited from the env.